clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/BaseKeyValueOperation.java

```diff
@@ -25,8 +25,10 @@
 
   private final K key;
   private final V value;
+  private final long timeStamp;
+  private final byte isFirst;
```

Comment:
Well, readability of code. It would make all logic cleaner but for codec methods. Also, you are adding a byte for a bit. Why not use the most significant bit of the timestamp since that one can never be negative?